src/etc/htmldocck.py

```diff
@@ -209,12 +177,7 @@ def get_commands(template):
 
             cmd = m.group("cmd")
             negated = m.group("negated") == "!"
-            if not negated and cmd in KNOWN_DIRECTIVE_NAMES:
-                continue
             args = m.group("args")
-            if args and not args[:1].isspace():
-                print_err(lineno, line, "Invalid template syntax")
-                continue
```

Comment:
In a roundabout way this used to check that the separator between the command and its args was a space.